src/PKSim.Core/Services/PKSimParameterStartValuesCreator.cs

```diff
@@ -184,7 +184,7 @@ private void setParameterValuesForStandardContainer(IndividualMolecule molecule,
       {
          var containerName = relativeExpressionContainerNameFrom(molecule, expressionContainer);
          var relExpPath = relExpPathFor(molecule, expressionContainer);
-         trySetValue(relExpPath, molecule.GetRelativeExpressionParameterFor(containerName));
+        //TODO  trySetValue(relExpPath, molecule.GetRelativeExpressionParameterFor(containerName));
```

Comment:
interesting also here. What do  we do in the PSV now. We need to discuss @Yuri05 and @PavelBal

Reply:
I think it's clear after our discussion yesterday?